static/js/core.js

```diff
@@ -50,7 +50,7 @@ $.extend({
                 return m[1].toUpperCase();
             });
         }
-        return (reg).test(MEDUSA.config[pyVar]);
+        return (reg).test(MEDUSA.config.layout[pyVar]);
```

Comment:
I'll check if it is, if it's used else where then we need another fix. I'll look at this tomorrow morning. 👍